app-emulation/containerd/containerd-9999.ebuild

```diff
@@ -13,7 +13,7 @@ if [[ ${PV} == *9999 ]]; then
 else
 	MY_PV="${PV/_rc/-rc.}"
 	EGIT_COMMIT="v${MY_PV}"
-	CONTAINERD_COMMIT="69a4a55aa42353c71aae40e5bb27a412d8e3149b"
+	CONTAINERD_COMMIT="d9768bd283183fc51cc587249530adc3fa43ebbb"
```

Comment:
I don't get where these commits come from. For both [69a4a55aa42353c71aae40e5bb27a412d8e3149b](https://github.com/containerd/containerd/commit/69a4a55aa42353c71aae40e5bb27a412d8e3149b) and [d9768bd283183fc51cc587249530adc3fa43ebbb](https://github.com/containerd/containerd/commit/d9768bd283183fc51cc587249530adc3fa43ebbb) I get 404 on github.

Reply:
```
$ git ls-remote https://github.com/containerd/containerd | tail -n4
69a4a55aa42353c71aae40e5bb27a412d8e3149b        refs/tags/v1.5.4
69107e47a62e1d690afa2b9b1d43f8ece3ff4483        refs/tags/v1.5.4^{}
d9768bd283183fc51cc587249530adc3fa43ebbb        refs/tags/v1.5.5
72cec4be58a9eb6b2910f5d10f1c01ca47d231c0        refs/tags/v1.5.5^{}
```
they are the hashes of the tags themselves.

Reply:
The commits are [generated](https://github.com/kinvolk/coreos-overlay/blob/38daddf87a34043b70ff5717cc35a9d437111948/.github/workflows/containerd-releases-main.yml#L18) by `git rev-parse VER`.

So for v1.5.5, under the containerd repo:

```
$ git rev-parse v1.5.5
d9768bd283183fc51cc587249530adc3fa43ebbb

$ git show d9768bd283183fc51cc587249530adc3fa43ebbb
tag v1.5.5
Tagger: Derek McGowan <derek@mcg.dev>
Date:   Thu Jul 29 10:03:37 2021 -0700

containerd 1.5.5

...

commit 72cec4be58a9eb6b2910f5d10f1c01ca47d231c0 (tag: v1.5.5)
Merge: 1c13c54c 677fade0
Author: Derek McGowan <derek@mcg.dev>
Date:   Thu Jul 29 10:02:27 2021 -0700

    Merge pull request #5805 from dmcgowan/prepare-1.5.5
    
    [release/1.5] Prepare release notes for v1.5.5

```

Long story short, `d9768bd283183fc51cc587249530adc3fa43ebbb` is equivalent to [72cec4be58a9eb6b2910f5d10f1c01ca47d231c0](https://github.com/containerd/containerd/commit/72cec4be58a9eb6b2910f5d10f1c01ca47d231c0) .